src/docs/automation-and-resiliency/openshift-project-resource-quotas.md

```diff
@@ -36,48 +36,16 @@ If the default allocations aren't sufficient for your application, [you can ask
 - [Storage quotas](#storage-quotas)
 - [Related pages](#related-pages)
 
-## CPU quotas
-
-The following CPU quotas are currently available on the platform. All CPU requests and limits are shown in cores (not millicores) and represent the maximum for the combined CPU request or CPU limit values for **all pods in a namespace**.
+## CPU request quotas
 
-```
-cpu-request-0.5-limit-1.5: CPU Request 0.5 core, CPU Limit 1.5 cores
-cpu-request-1-limit-2: CPU Request 1 core, CPU Limit 2 cores
-cpu-request-2-limit-4: CPU Request 2 cores, CPU Limit 4 cores
-cpu-request-4-limit-8: CPU Request 4 cores, CPU Limit 8 cores
-cpu-request-8-limit-16: CPU Request 8cores, CPU Limit 16 cores
-cpu-request-16-limit-32: CPU Request 16 cores, CPU Limit 32 cores
-cpu-request-32-limit-64: CPU Request 32 cores, CPU Limit 64 cores
-```
+CPU request quotas can be set to the desired level via the [Platform Product Registry](https://registry.developer.gov.bc.ca/login).  All CPU requests quotas are shown in cores (not millicores) and represent the total **maximum** for the combined CPU request **all pods in a namespace**.
```

Comment:
There are two quotas, one for long-running pods, and one for short running pods like jobs, builds, and DeploymentController deploy pods. So "all pods in a namespace" is a little incorrect since pods will match one or the other quota. In theory, you can use "double" the quota by having as many Job pods as long running pods.

Reply:
Maybe that's all a little to nuanced though? It might be simpler and clearer to just say "all pods"?

Reply:
Interesting, the **all pods** callout wasn't part of this change, but maybe there's a better way to say it. 

I might just add a line to say 'these totals are further broken down into separate quotas for short and long-running pods'. 

Would that cover improve it @StevenBarre ?

Reply:
Yup, i think that would be great.

Reply:
Ok, I've added this and some brief explanations of long-running, short running and best-effort pods and the quotas that apply to each.

Reply:
Perfect